### DISCUSSION TITLE
Change a heading from left aligned to center aligned

### DISCUSSION THREAD

**Discussion by KennethGottfredsen** [quarto-dev/quarto-cli#3857] [Q&A] [answered]
Hi guys, 

Is it possible to center a standard heading in Quarto? I'm knitting to pdf. 

/ Kenneth.

**Comment by mcanouil** [comment, marked as answer]
Could you provide a small Quarto document and detail what you call "heading"?

**Reply by KennethGottfredsen** [reply]
I mean a section heading. By default it's left aligned. I want it to be centered when I'm rendering it as a PDF-document. Below is an example: 
 
`
# Kotters forandringsmodel

`

**Reply by mcanouil** [reply]
Where is the example Quarto document?

Meantime, because it's a LaTeX question rather than a Quarto question, the following should get you going: <https://duckduckgo.com/?q=center+align+section+latex>

**Reply by KennethGottfredsen** [reply]
Thank you!

**Comment by wwhuang80** [comment]
I ran into the same issue. Maybe because Quarto uses KOMAscript, some of the solutions available (such as using sectsty) gave me some side problems. Adding
`\addtokomafont{section}{\centering} `
command to the Quarto preamble seems to be one of the best options. [tex.stackexchange.com link](https://tex.stackexchange.com/questions/292727/how-to-change-the-font-style-of-elements-in-koma-script)
